repository/spark/operator/templates/spark-operator-deployment.yaml

```diff
@@ -67,11 +67,7 @@ spec:
         {{ end }}
         args:
         - -v={{ .Params.logLevel }}
-        {{- if (ne .Params.sparkJobNamespace "") }}
```

Comment:
shell we keep the conditional or empty value works well?

Reply:
Yes, I’ve checked and can confirm empty value works, so we can pass the parameter value directly to the operator